### ISSUE TITLE
ANDI testing: Contrast ratios in Dark mode

### ISSUE THREAD

**Issue by JenM00re** [Swarthmore/accessible-docs#19]
It appears as though the contrast ratios in Dark Mode for the course titles could be lacking. ANDI gives a suggested color.

![ANDI Testing Dark Mode Contrast Ratio Course Title](https://github.com/Swarthmore/a11y-doc-repo/assets/91497451/0b9debf3-2a1f-4109-80b5-0b1a36f0fabe)

Explanation:
- [Alert: Contrast ratio minimum requirement](https://www.ssa.gov/accessibility/andi/help/alerts.html?min_contrast)
- [Contrast Checker | WebAIM](https://webaim.org/resources/contrastchecker/)
